### PR TITLE
Add French translation

### PR DESCRIPTION
**Pull request by locness3**
Thanks for the great app! Here's a French translation.

### CODE REVIEW

**Comment by msoultanidis** [comment]
Thanks for taking the time to translate Quillnote! 
It seems like you created your branch from the `main` branch. Development happens in the `develop` branch and currently there are some new strings there. Please take a look at it when you have the time.

**Comment by locness3** [comment]
Oops, I used the file from the main branch, then switched to develop, then commited it. I'll have a look.

**Comment by locness3** [comment]
I added the new strings!

File: app/src/main/res/values-fr/strings.xml [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+    <string name="preferences_nextcloud_instance_url">Adresse du server Nextcloud</string>
+    <string name="preferences_nextcloud_set_your_credentials">Définissez vos identifiants</string>
+    <string name="preferences_nextcloud_set_server_url">Définissez l\'URL du serveur</string>
+    <string name="preferences_sync_on_wifi">Wi-Fi</string>
+    <string name="preferences_sync_on_wifi_data">Wi-Fi ou données mobiles</string>
+    <string name="preferences_sync_when_on">Synchroniser avec la connexion</string>
+    <string name="preferences_background_sync">Synchroniser en arrière-plan</string>
+    <string name="preferences_background_sync_enabled">Activé</string>
+    <string name="preferences_background_sync_disabled">Désactivé</string>
+    <string name="preferences_new_notes_synchronizable">New notes synchronizable</string>
```
Comment (locness3) [inline app/src/main/res/values-fr/strings.xml:86]: I should point out that I didn't understand this string, I didn't translate it.

Comment (msoultanidis) [reply app/src/main/res/values-fr/strings.xml:86]: Notes have an attribute called "isLocalOnly" which dictates whether changes to that note should be synced remotely or not. Users can change that inside the editor if they tick the "Do not sync" menu item. This preference is about whether newly created notes are synchronizable or not by default. For example, if  users set that preference to "No" then all newly created notes will have the "Do not sync" menu item ticked by default.

Comment (locness3) [reply app/src/main/res/values-fr/strings.xml:86]: Should be changed to "Synchronise new notes" or smth

Comment (msoultanidis) [reply app/src/main/res/values-fr/strings.xml:86]: Indeed, it's not clear enough currently.

Comment (locness3) [reply app/src/main/res/values-fr/strings.xml]: I added the translation for that string.

**Comment by msoultanidis** [comment]
Could you please translate some more strings I just added? 

Thanks for your time!

**Comment by locness3** [comment]
Done.

**Comment by msoultanidis** [comment]
Looks good, merged!

**Pull request merged**